privacy.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>
+			mulcahyfamily.org Chrome Extensions' Privacy Policies
+		</title>
+	</head>
+	<body>
+        	<h1>
```

Comment:
This bit has to go - I left Yelp a few weeks ago and the extension has been re-homed (off my account).